<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действие</string>
    </property>
    <addaction name="add"/>
    <addaction name="remove"/>
    <addaction name="doLists"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="add"/>
   <addaction name="remove"/>
   <addaction name="doLists"/>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/1024x1024bb.png</normaloff>:/new/prefix1/icons/1024x1024bb.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/png-clipart-computer-icons-icon-design-iconfinder-save-icon-miscellaneous-saving.png</normaloff>:/new/prefix1/icons/png-clipart-computer-icons-icon-design-iconfinder-save-icon-miscellaneous-saving.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/png-transparent-green-and-white-icon-grass-symbol-green-apps-dialog-add-flag-canada-rectangle.png</normaloff>:/new/prefix1/icons/png-transparent-green-and-white-icon-grass-symbol-green-apps-dialog-add-flag-canada-rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Добавить слово</string>
   </property>
  </action>
  <action name="remove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/i.webp</normaloff>:/new/prefix1/icons/i.webp</iconset>
   </property>
   <property name="text">
    <string>Удалить слово</string>
   </property>
  </action>
  <action name="doLists">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/icons/png-clipart-computer-icons-shopping-list-others-angle-text.png</normaloff>:/new/prefix1/icons/png-clipart-computer-icons-shopping-list-others-angle-text.png</iconset>
   </property>
   <property name="text">
    <string>Сформировать списки слов</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
